filterizePy/test/test_mirror.py

```diff
@@ -7,47 +7,55 @@
 # This script tests the function from mirror.py.
 
 # This script tests the mirror function of the filterizePy package.
+import sys
+sys.path.append('../../')
 
 from skimage import io
-import pytest
 import numpy as np
-from filterize import mirror
+import pytest
+from filterizePy.mirror import mirror
 
-input_img = io.imread('../img/input_image.png')
+input_img = "../../img/test_original.jpg"
 mirror(input_img)
-output_img = io.imread('../img/mirror_img.png')
+input_img = io.imread('../../img/test_original.jpg')
+output_img = io.imread('../../img/mirrored_test_original.jpg')
 
 def test_flip_same_size(input_img, output_img):
     # checks input and output size
     assert input_img.shape == output_img.shape, "Input and output dimensions do not match"
 
 def test_flip_column_flip(input_img, output_img):
     # first and last column should have the same pixels
-    assert np.array_equal(input_img[:,0,:], output_img[:,-1,:]), "First and last pixel columns do not match"
+    assert np.all(np.abs(input_img[:,0,:] - output_img[:,-1,:]) < 20), "First and last pixel columns do not match"
 
 def test_flip_column_flip_mid(input_img, output_img):
     # middle column should stay the same, the cases are different for even and odd dimensioned images
     m,n,d = input_img.shape
+    input_img = np.array(input_img, dtype=np.float64)
+    output_img = np.array(output_img, dtype=np.float64)
 
     if n % 2 == 0:
         mid = n//2
-        assert np.array_equal(input_img[:,mid,:], output_img[:,mid-1,:])
+        assert np.all(np.abs(input_img[:,mid,:] - output_img[:,mid-1,:]) < 20)
     else:
         mid = n//2
-        assert np.array_equal(input_img[:,mid,:], output_img[:,mid,:]), "Error in middle transform column"
+        assert np.all(np.abs(input_img[:,mid,:] - output_img[:,mid,:]) < 20)
 
-def test_flip_file_path():
+def reversed_input_img():
     with pytest.raises(FileNotFoundError):
         mirror("not a file path")
 
 def test_flip_invalid_input():
     with pytest.raises(AttributeError):
         mirror(123)
 
-def test_flip_invalid_input():
+def test_flip_invalid_input_2():
     with pytest.raises(TypeError):
         mirror()
 
-def test_flip_invalid_input():
-    with pytest.raises(SyntaxError):
-        mirror(,)
+reversed_input_img()
```

Comment:
Hi Jack, 
I really like the if-else function. 

You can delete the code lines below and use `python -m pytest test/test_mirror.py` to run your test in bash.
Don't forget to rename your test functions to begin with "test_".

Good job over all! <3

Chao

Reply:
Thanks Chao. Resolved.